themes/docsy/layouts/partials/page-meta-links.html

```diff
@@ -5,9 +5,9 @@
 {{ if $gh_repo }}
 <div class="td-page-meta ml-2 pb-1 pt-2 mb-0">
 {{ $editURL := printf "%s/edit/master/content/%s" $gh_repo .Path }}
-{{ if and ($gh_subdir) (.Site.Language.Lang) }}
+{{ if and ($gh_subdir) (.Site.IsMultiLingual) }}
 {{ $editURL = printf "%s/edit/master/%s/content/%s/%s" $gh_repo $gh_subdir ($.Site.Language.Lang) $.Path }}
-{{ else if .Site.Language.Lang }}
+{{ else if .Site.IsMultiLingual }}
```

Comment:
Can you instead put a copy of this file with your updates in layouts/partials/page-meta-links.html?

Reply:
I updated as requested. But if `docsy` fixes this in the future, we may need to delete this file.